<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameFieldWidget</class>
 <widget class="QWidget" name="GameFieldWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000
</string>
  </property>
  <widget class="QPushButton" name="ToMenuButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>800</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Images/Images/ToMenuButton.png); border-style: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="HelmetLabelWidget">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>800</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/HelmetLabelWidget.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ArmorLabelWidget">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>800</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/ArmorLabelWidget.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PantsLabelWidget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>800</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/PantsLabelWidgwet.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BootsLabelWidget">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>800</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/BootsLabelWidget.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WeaponLabelWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>800</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: url(:/Images/Images/WeaponLabelWidget.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ScoreLineWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>800</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661; border-style: #000000;</string>
   </property>
   <property name="text">
    <string>0 P</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="HealthLineWidget">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>800</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661
; border-style: #000000;</string>
   </property>
   <property name="text">
    <string>10HP</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>800</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661
; border-style: None;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>800</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #666661
; border-style: None;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
